### ISSUE TITLE
Simulator arguments

### ISSUE THREAD

**Issue by jgosmann** [tcstewar/pytry#19]
There doesn't seem any possibility to pass arguments to the simulator. This would be useful to turn off the optimizer for example or specify the context for Nengo OCL.

**Comment by tcstewar** [comment]
Yup...  I haven't been able to think of a good way of doing this.   When I've needed to, I've just switched to doing a `pytry.PlotTrial` rather than a `pytry.NengoTrial` and manually created the `Simulator` myself.....

**Comment by tcstewar** [comment]
Never mind!  I just figured out a nice way to do this.  :)  You can now do `"--backend nengo_whatever(myarg=3)"` and it'll pass those parameters on to the simulator.  :)

**Comment by tcstewar** [comment]
Here's the commit:  205830e06dd4ab08089763b2392276f170659693
